runtime/src/main/java/org/corfudb/runtime/object/transactions/TransactionalContext.java

```diff
@@ -27,6 +30,25 @@
     private static final ThreadLocal<Deque<AbstractTransactionalContext>> threadStack = ThreadLocal.withInitial(
             LinkedList<AbstractTransactionalContext>::new);
 
+    /** A navigable set (priority queue) of contexts. The minimum context
```

Comment:
this comment is about the lines above it, which are not modified in this PR, but it's time to change:

threadStack is a bad name. It should be threadTransactionStack or something like that